modules/cluster-logging-visualizer-indices.adoc

```diff
@@ -39,8 +39,8 @@ To define index patterns and create visualizations in Kibana:
 
 . Create your Kibana index patterns by clicking *Management* -> *Index Patterns* -> *Create index pattern*:
 
-** Users must manually create index patterns to see logs for their projects. Users should create a new index pattern named *app* and use the `@timestamp` time field to view their container logs. 
+** Each user must manually create index patterns when logging into Kibana the first time in order to see logs for their projects. Users must create an index pattern named 'app' and use the `@timestamp` time field to view their container logs.
```

Comment:
These should all be backticks instead of single quotes. Please update this line and the following one.